.eslintrc.js

```diff
@@ -91,7 +91,6 @@ module.exports = {
 		'react/jsx-no-undef': 2,
 		'react/jsx-no-duplicate-props': 1,
 		'react/react-in-jsx-scope': 2,
-		'react/no-danger': 2,
```

Comment:
`dangerouslySetInnerHTML` is so cumbersome to use (on purpose) that it's highly unlikely to be used accidentally. I think this rule only adds noise.